test/client/fixtures/sandbox/node/dom-processor-test.js

```diff
@@ -486,7 +488,9 @@ test('remove the "integrity" attribute from the link and script tags (GH-235)',
     domProcessor.processElement(link, urlReplacer);
 
     ok(!script.hasAttribute('integrity'));
+    strictEqual(nativeMethods.getAttribute.call(script, storedIntegrityAttrName), integrityValue);
```

Comment:
```js
strictEqual(script.getAttribute('integrity'), integrityValue);
strictEqual(nativeMethods.getAttribute.call(script, 'integrity'), null);
```
Instead of this assertion.